8.x/theme-development/basics.md

```diff
@@ -2,4 +2,190 @@
+
+3- The end of the file name is **_mobile** or **_app**.
+
+![blade-file-3](/blade-file-3.png)
+
+As you know, responsive web design works through CSS, using various settings to different style properties depending on the screen size of the user’s device. Sometimes the design structure of pages on mobile and desktop is different, so create a file with the **_mobile** extension to be displayed on mobile pages with this structure. And for mobile applications, the system provides **_app** pages.
+
+Blade pages are reviewed below. Some pages are required and should be in the template, but some pages are optional.
+
+## About page
+![about](/about.png)
+
+This page is optional and you can create this page if needed on the project.
+
+## Address page
+![address](/address.png)
+
+These pages must be created and In these pages, there is a form with post method and inputs: state_id, city_id, superscription, phone_number, zipcode, transferee_name. And be sure to add  **csrf_field** inside the form.
+
+These inputs are required and if one of the inputs is not filled, there is an error when sending to the **customer.address.store** rout. Form information should not be changed if the form design is changed.
+
+## Auth page
+![auth](/auth.png)
+
+To authenticate in the system:
+
+First, enter the phone number or email in the **auth-mobile-show.blade.php** or **auth-email-show.blade.php** page.
+
+If a member of the system and the password is set, the **auth-mobile-password.blade.php** page is displayed. And if the password is not set, entering the one-time code in the **auth-mobile-check.blade.php** or **auth-email-check.blade.php** page.
+
+If not a member of the system, entering the one-time code in the **auth-mobile-check.blade.php** or **auth-email-check.blade.php** page. and enter the **auth-email-register.blade.php** or **auth-mobile-register.blade.php** page.
+
+Let's check the authentication files:
+
+1- **auth-mobile-show.blade.php**
+
+In this page, there is an input with **type ="mobile"** and **name ="mobile"** that this email will be sent to the backend with the **customer-auth.do-mobile-auth** route. 
+
+2- **auth-mobile-check.blade.php**
+
+There is an input, enter the one-time code to send to the server.
+
+3- **auth-mobile-password.blade.php**
+
+There is an input with name = "password"  enter the password and then login.
+
+4- **auth-mobile-register.blade.php**
+
+In this page, there is a form with post method and inputs: name, family, email, phone, main_phone, national_code. And be sure to add csrf_field inside the form.
+
+These pages must be created and the inputs inside the pages are mandatory and should not be changed. Only the design of the pages can be changed.If there are changes in the structure of ecommerce, it will be announced in the documents of the next version.
+
+## Blog page
+![blog](/blog.png)
+
+**blog-list.blade.php** page is required and must be created.
```

Comment:
Please don't mention this note in every section. Make a section named `Required blade files` and underneath that section tell the audience that these files are needed by the system and you have to place them in your project.
Other files like partials and custom pages are not necessary and you can add them on demand.
Please avoid writing repeated texts all over the document.